src/test/regress/expected/subquery_view.out

```diff
@@ -559,9 +560,80 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT user_id, user
        3 |       6 | Thu Nov 23 14:00:13.20013 2017  |          3 |       3 |       3 |
 (5 rows)
 
+CREATE TABLE reference_table (text_col text, int_col int);
+SELECT create_reference_table('reference_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
 SET client_min_messages TO DEFAULT;
+EXPLAIN (COSTS OFF) WITH cte AS (
+  SELECT application_name AS text_col
+  FROM pg_stat_activity
+) SELECT * FROM reference_table JOIN cte USING (text_col);
+                                            QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (Citus Adaptive)
+   ->  Distributed Subplan XXX_1
+         ->  Function Scan on pg_stat_get_activity s
+   ->  Distributed Subplan XXX_2
+         ->  Custom Scan (Citus Adaptive)
+               Task Count: 1
+               Tasks Shown: All
+               ->  Task
+                     Node: host=localhost port=xxxxx dbname=regression
+                     ->  Hash Left Join
+                           Hash Cond: (intermediate_result.usesysid = u.oid)
+                           ->  Hash Left Join
+                                 Hash Cond: (intermediate_result.datid = d.oid)
+                                 ->  Function Scan on read_intermediate_result intermediate_result
+                                 ->  Hash
+                                       ->  Seq Scan on pg_database d
```

Comment:
I assume this subplan runs on the coordinator and that's why we can query pg_Database here, but that's not obvious from the explain plan. Is there a way to verify that?

Reply:
only way I can think of is enabling debug messages but even debug1 messages seem to differ in this test file for different pg versions  :/
f0d56b6

Reply:
I guess you can temporarily restore client_min_messages for the CREATE TABLE.

Reply:
Actually problem is with explain commands,

I was having diffs when I enabled debug1 before explain commands:
https://app.circleci.com/pipelines/github/citusdata/citus/12081/workflows/a347a91e-1d7d-4072-920d-2f2649f5a183/jobs/199030

```diff
 EXPLAIN (COSTS OFF) WITH cte AS (
   SELECT application_name AS text_col
   FROM view_on_views
 ) SELECT * FROM reference_table JOIN cte USING (text_col);
-DEBUG:  generating subplan XXX_1 for CTE cte: SELECT application_name AS text_col FROM (SELECT pg_stat_activity.application_name, pg_locks.pid FROM (SELECT s.datid, d.datname, s.pid, s.usesysid, u.rolname AS usename, s.application_name, s.client_addr, s.client_hostname, s.client_port, s.backend_start, s.xact_start, s.query_start, s.state_change, s.wait_event_type, s.wait_event, s.state, s.backend_xid, s.backend_xmin, s.query, s.backend_type FROM ((pg_stat_get_activity(NULL::integer) s(datid, pid, usesysid, application_name, state, query, wait_event_type, wait_event, xact_start, query_start, backend_start, state_change, client_addr, client_hostname, client_port, backend_xid, backend_xmin, backend_type, ssl, sslversion, sslcipher, sslbits, sslcompression, ssl_client_dn, ssl_client_serial, ssl_issuer_dn, gss_auth, gss_princ, gss_enc) LEFT JOIN pg_database d ON ((s.datid OPERATOR(pg_catalog.=) d.oid))) LEFT JOIN pg_authid u ON ((s.usesysid OPERATOR(pg_catalog.=) u.oid)))) pg_stat_activity, (SELECT l.locktype, l.database, l.relation, l.page, l.tuple, l.virtualxid, l.transactionid, l.classid, l.objid, l.objsubid, l.virtualtransaction, l.pid, l.mode, l.granted, l.fastpath FROM pg_lock_status() l(locktype, database, relation, page, tuple, virtualxid, transactionid, classid, objid, objsubid, virtualtransaction, pid, mode, granted, fastpath)) pg_locks) view_on_views
+DEBUG:  generating subplan 56_1 for CTE cte: SELECT application_name AS text_col FROM (SELECT pg_stat_activity.application_name, pg_locks.pid FROM (SELECT s.datid, d.datname, s.pid, s.leader_pid, s.usesysid, u.rolname AS usename, s.application_name, s.client_addr, s.client_hostname, s.client_port, s.backend_start, s.xact_start, s.query_start, s.state_change, s.wait_event_type, s.wait_event, s.state, s.backend_xid, s.backend_xmin, s.query, s.backend_type FROM ((pg_stat_get_activity(NULL::integer) s(datid, pid, usesysid, application_name, state, query, wait_event_type, wait_event, xact_start, query_start, backend_start, state_change, client_addr, client_hostname, client_port, backend_xid, backend_xmin, backend_type, ssl, sslversion, sslcipher, sslbits, sslcompression, ssl_client_dn, ssl_client_serial, ssl_issuer_dn, gss_auth, gss_princ, gss_enc, leader_pid) LEFT JOIN pg_database d ON ((s.datid OPERATOR(pg_catalog.=) d.oid))) LEFT JOIN pg_authid u ON ((s.usesysid OPERATOR(pg_catalog.=) u.oid)))) pg_stat_activity, (SELECT l.locktype, l.database, l.relation, l.page, l.tuple, l.virtualxid, l.transactionid, l.classid, l.objid, l.objsubid, l.virtualtransaction, l.pid, l.mode, l.granted, l.fastpath FROM pg_lock_status() l(locktype, database, relation, page, tuple, virtualxid, transactionid, classid, objid, objsubid, virtualtransaction, pid, mode, granted, fastpath)) pg_locks) view_on_views
```

Reason was us generating different subplans in different pg versions.